docs/how_to_guides/pushing_and_pulling.mdx

```diff
@@ -72,7 +72,7 @@ In this example, we'll push a collection of files.
 - A single file (`artifact.txt`) that represents overview content that might be displayed as a repository overview
 - A collection of files (`docs/*`) that represents detailed content. When specifying a directory, ORAS will automatically tar the contents.
 
-See [OCI Artifacts][artifacts] for more details.
+See [OCI Artifacts](https://github.com/opencontainers/artifacts/blob/main/artifact-authors.md#defining-a-unique-artifact-type) for more details.
```

Comment:
```suggestion
See [OCI Artifacts](https://github.com/opencontainers/image-spec/blob/main/manifest.md#guidelines-for-artifact-usage) for more details.
```

Reply:
https://github.com/opencontainers/artifacts is archived so we need to point it to a new repository.